java/core/src/main/java/com/microsoft/bond/Bonded.java

```diff
@@ -19,7 +19,8 @@ public Bonded(T instance) {
 
     @Override
     public void serialize(ProtocolWriter writer) throws IOException {
-        value.serialize(writer);
+        //value.serialize(writer);
```

Comment:
See Chad's comment earlier. Added a TODO.

---
In reply to: [123404614](https://github.com/Microsoft/bond/pull/502#discussion_r123404614) [](ancestors = 123404614)